_drafts/SC3-1-2-lang.md

```diff
@@ -0,0 +1,76 @@
+---
+name: Validity of HTML Attribute with in elements of page body
+
+description:
+- This rule checks that `lang` and/or `xml:lang` attribute on elements with in the `body` of a web page are correct.
```

Comment:
Maybe "correct" is too broad a term? This could imply that the rule should also check of it has been correctly applied to the content. 
Consider aligning it with the description in https://github.com/auto-wcag/auto-wcag/pull/109/files